cloud/azure/main.tf

```
@@ -206,6 +119,55 @@ resource "azurerm_log_analytics_workspace" "civiform_logs" {
   retention_in_days   = 30
 }
 
+output "app_service_default_hostname" {
+  value = "https://${azurerm_app_service.civiform-app.default_site_hostname}"
+}
+
+resource "azurerm_monitor_diagnostic_setting" "app_service_log_analytics" {
+  name                       = "app_service_log_analytics"
```

suggestion (blocking): 

```suggestion
  name                       = "${var.application_name}_log_analytics"
```

---

fixed